File: WineryAPI/Models/Magacin.cs
using System;
using System.Collections.Generic;

namespace WineryAPI.Models;

public partial class Magacin
{
    public int Idmag { get; set; }

    public string Nazivmag { get; set; } = null!;

    public int Kapacitetmag { get; set; }

    public decimal Tempmag { get; set; }

    public virtual ICollection<Boca> Bocas { get; set; } = new List<Boca>();
}

File: winery-frontend/src/types/magacin.ts
export interface Magacin {
  idmag: number
  nazivmag: string
  kapacitetmag: number
  tempmag: number
  brojBoca: number
  popunjenost: number
}

export interface MagacinDetail {
  idmag: number
  nazivmag: string
  kapacitetmag: number
  tempmag: number
  brojBoca: number
  popunjenost: number
}

export interface CreateMagacinRequest {
  nazivmag: string
  kapacitetmag: number
  tempmag: number
}

export interface UpdateMagacinRequest {
  nazivmag: string
  kapacitetmag: number
  tempmag: number
}

